src/build-data/buildh.in

```diff
@@ -210,20 +210,26 @@
 #define BOTAN_SYSTEM_RNG_POLL_TIMEOUT_MS 20
 
 /*
-How many times to read from the RDRAND/RDSEED RNGs.
-Each read generates 32 bits of output
+* How many bytes of output to read from RDRAND for seeding purposes.
+* We default to 8192 bytes which is (according to Intel documentation)
+* guaranteed to cause at least one reseeding of the internal RNG.
 */
-#define BOTAN_ENTROPY_INTEL_RNG_POLLS 32
+#define BOTAN_ENTROPY_INTEL_RDRAND_BYTES 8192
```

Comment:
The idea re 8192 was that 1022*8 < 8192 but this seems to only apply for specifically the 64-bit RDRAND which we can't use on 32-bit. Maybe it is better to use the sleeping approach. But, not clear from the doc if the sleep approach works if there is another concurrent user of RDRAND since in that case there is no idle period.

Reply:
I did some experimentation on a Skylake system. Doing 32 RDRANDs with sleep took ~2 ms and produced 1024 bits of output. Also it is still unclear to me if this works if there is another concurrent RDRAND user. Reading 8192 bytes (512 samples) takes .1 ms and produces 64x as much output. I was concerned that this only worked correctly with 64-bit RDRAND but from other parts in the docs it is clear that any read size works "There is an upper bound of 511 samples per seed in the implementation where samples are 128 bits in size and can provide two 64-bit random numbers each. "